docs/usage/dependency-callables.md

```diff
@@ -34,3 +34,12 @@ Get the current superuser. Will throw a `401 Unauthorized` if missing or wrong c
 def protected_route(user: User = Depends(fastapi_users.get_current_superuser)):
     return f'Hello, {user.email}'
 ```
+
```

Comment:
Could you add a subsection (`##`) named "In path operation"? Could be interesting also to link the related page in FastAPI documentation: https://fastapi.tiangolo.com/tutorial/dependencies/dependencies-in-path-operation-decorators/